src/test/scala/com/databricks/sql/package.scala

```diff
@@ -0,0 +1,24 @@
+package com.databricks.sql
+
+import org.apache.spark.sql.Column
+import org.apache.spark.sql.functions.lit
+
+object functions {
```

Comment:
@edurdevic can we just add here a comment explicitly stating why we added this object.
I know it is rather obvious from the dummy function definitions, but still it is better if we added that comment